interactive_engine/compiler/src/test/resources/config/gs_interactive.yaml

```diff
@@ -0,0 +1,35 @@
+---
+version: 0.0.1
+directories:
+  workspace: .
+  subdirs:
+    data: target/test-classes/config  # by default data, relative to ${workspace}
+    conf: conf # by default conf, relative to ${workspace}
+    logs: logs  # by default logs, relative to ${workspace}
+log_level: INFO # default INFO
+default_graph: modern  # configure the graph to be loaded while starting the service, if graph name not specified
+# may include other configuration items of other engines
+compute_engine:
+  type: pegasus  # [hiactor|pegasus] hiactor is for high-qps scenario, pegasus is for bi scenario
```

Comment:
type is `hiactor`?

Reply:
fix in https://github.com/alibaba/GraphScope/pull/3141/commits/8d266975627bc0cda07c364bb0b0644358fc700d